modules/op-running-as-root-user-namespace.adoc

```diff
@@ -0,0 +1,176 @@
+// Module included in the following assemblies:
+//
+// * cicd/pipelines/running-workloads-and-buildah-as-user-namespaces-on-openshift-pipelines.adoc
+:_content-type: PROCEDURE
+
+[id="op-running-as-root-user-namespace_{context}"]
+= Running buildah as root in a user namespace
+
+If you are using {pipelines-shortname} to run pods in the user namespace, you must pass the annotations that the CRI-O container engine expects. You can add these annotations for all users, or for a specific namespace.
+
+.Procedure
+
+. Export the `buildah` cluster task as a task to the project's namepsace.
++
+[source,terminal]
+----
+$ tkn task create --from=buildah
+----
+
+. Edit the `buildah` task within the project's namespace:
++
+[source,terminal]
+----
+$ oc edit task buildah
+----
+
+. Add annotations to the task:
++
+[source,yaml]
+----
+...
+annotations:
+  io.kubernetes.cri-o.userns-mode: "auto"
+  io.openshift.builder: "true"
+...
+----
+
+. Verify that the `buildah` task's container is running as root in the user-defined namespace. 
```

Comment:
user doesn't have to go on the node to activate non-root, it's ony useful if as an admin you need to make sure the setup works prroperly